packages/app-desktop/services/autoUpdater/AutoUpdaterService.ts

```diff
@@ -0,0 +1,120 @@
+import { app, BrowserWindow } from 'electron';
+import { autoUpdater, UpdateInfo } from 'electron-updater';
+import log from 'electron-log';
+import path = require('path');
+import { setInterval } from 'timers';
```

Comment:
Please `shim.setInterval` if possible - the reason is that it's going to use a more reliable timer and it's also easier to track all timers in the app.